gp-includes/formats/format-android.php

```diff
@@ -161,7 +161,7 @@ private function unescape( $string ) {
 	 */
 	protected function escape( $string ) {
 		$string = addcslashes( $string, "'\n" );
-		$string = str_replace( array( '&', '<' ), array( '&amp;', '&lt;' ), $string );
```

Comment:
The escaping should be done on the line above (`addcslashes()`).

In addition, do we need to escape `\` as well?